pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -713,14 +713,20 @@ func podNetworkRequiredStatusCause(field *k8sfield.Path) metav1.StatusCause {
 }
 
 func validateLiveMigration(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, config *virtconfig.ClusterConfig) (causes []metav1.StatusCause) {
-	if !config.LiveMigrationEnabled() && spec.EvictionStrategy != nil {
+	evictionStrategy := config.GetConfig().EvictionStrategy
+
+	if spec.EvictionStrategy != nil {
+		evictionStrategy = spec.EvictionStrategy
+	}
+	if !config.LiveMigrationEnabled() && evictionStrategy != nil {
```

Comment:
If `*evictionStrategy == v1.EvictionStrategyNone` then this shouldn't be an error. I suggest replacing to:
`if !config.LiveMigrationEnabled() && *evictionStrategy == v1.EvictionStrategyLiveMigrate {`

Reply:
Thanks! I did `if !config.LiveMigrationEnabled() && evictionStrategy != nil && *evictionStrategy != v1.EvictionStrategyNone` to be a tiny bit future-proof.